mgmtworker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Run from source root (directory has to contain cloudify-common, cloudify-agent, cloudify-manager)
+
+FROM python:3.10.6
+
+RUN apt-get update
+RUN pip install --upgrade pip
+
+WORKDIR /usr/src/app
```

Comment:
Why `src`? It's the app itself, not intended to be used as source, but for running.
I think the convention of putting things in /opt has served us well so far